audio_video_recorder/README.md

```diff
@@ -0,0 +1,78 @@
+# audio_video_recorder
+
+ROS package for recording audio and video synchronously
+
+![[Full video on Google Drive](https://drive.google.com/file/d/1TWnRKbOdq6jPza82eNhhjn56lQXRxWjl/view?usp=sharing)
+](.media/pr2_sample.gif)
+
+[Full video on Google Drive](https://drive.google.com/file/d/1TWnRKbOdq6jPza82eNhhjn56lQXRxWjl/view?usp=sharing)
+
+## Sample
+
+You can record audio and video on your laptop.
+
+```bash
+roslaunch audio_video_recorder sample_audio_video_recorder.launch
+```
+
+## Parameters
+
+Node: `audio_video_recorder/audio_video_recorder`
+
+### Common parameters
+
+- `queue_size` (`Int`, default: `100`)
+
+  Queue size
+
+- `do_timestamp` (`Bool`, default: `true`)
+
+  Do timestamp or not (`false` is not working now.)
+
+- `file_location` (`String`, default: `/tmp/test.avi`)
+
+  Output file location
+
+- `file_format` (`String`, default: `avi`)
+
+  Output file format (Only `avi` is supported now.)
+
+### Audio parameters
+
+- `audio_format` (`String`, default: `mp3`)
```

Comment:
why you need to store mp3, https://en.wikipedia.org/wiki/Audio_Video_Interleave says avi can contain both audio and video data

Reply:
This is `format` of subscribing audio topic.
We need to specify it first to create the gstreamer pipeline correctly.

Reply:
I see, so how about categorize parameters as recording vs subscribe, not audio vs video.  
- Common parameters for both subscribing and recording 
- Parameters for recording
  - video parameters for recording
  - audio parameters for recording
- Parameters for subscribing
  - parameters for image (and? or ? sensor_msgs/Image) subscribing
  - parameters for audio_common/??? subscribing
  -

Reply:
Thank you. That is a good idea. I will update the readme!